tests/post/steps/test_volume.py

```diff
@@ -373,6 +379,34 @@ def _check_file_content():
         name='checking content of {} on Pod {}'.format(path, name)
     )
 
+
+@then(parsers.parse("the backing storage for Volume '{name}' is created"))
+def check_storage_is_created(context, host, name):
+    volume = context.get(name)
+    assert volume is not None, 'volume {} not found in context'.format(name)
+    assert 'sparseLoopDevice' in volume['spec'], 'unsupported volume type'
```

Comment:
Worth mentioning in the error message that it's unsupported _for this test_.

Reply:
Done